linker/Linker/Driver.cs

```diff
@@ -346,6 +351,7 @@ static void Usage (string msg)
 			Console.WriteLine ("   --about             About the {0}", _linker);
 			Console.WriteLine ("   --version           Print the version number of the {0}", _linker);
 			Console.WriteLine ("   --skip-unresolved   Ignore unresolved types, methods, and assemblies (true or false)");
+			Console.WriteLine ("   --log-warnings      Log warnings for internal exceptions");
```

Comment:
If I'm reading the code correctly turning on this option will enable all LogMessage output, not just warnings. I think both the option name and the description are misleading. Maybe --log-messages or --output-log?

Reply:
They are a bit misleading. I changed them to LogMessages and --log-messages.